client/apps/articles_list/components/listView.jsx

```diff
@@ -0,0 +1,147 @@
+import PropTypes from 'prop-types'
+import React, { Component } from 'react'
+import { bindActionCreators } from 'redux'
+import { connect } from 'react-redux'
+import * as Actions from 'client/actions/articlesActions'
+import { data as sd } from 'sharify'
+
+import $ from 'jquery'
+import { debounce } from 'lodash'
+import FilterSearch from 'client/components/filter_search/index.coffee'
+import request from 'superagent'
+
+require('jquery-on-infinite-scroll')
+const query = require('../query.coffee')
+const icons = () => require('../icons.jade')(...arguments)
+
+export class ArticlesListView extends Component {
```

Comment:
I would make sure to map component name to file-name 1:1. At some point Positron snake_casing should go away in favor of more idiomatic modern JS naming (and for consistency across other Artsy apps); e.g.,
 
```javascript
import { ArticlesListView } from 'components/ArticlesListView'` 
```

(or something similar 😄)

But in the meantime file should still match export (`articles_list_view.js`) so that its clear what is being used where. 

cc @eessex

Reply:
👍 yeah ArticlesList is probably a better choice

Reply:
That's precisely what I renamed the class and file to, just haven't pushed it yet 😊